tests/solana_tests/runtime_errors.rs

```diff
@@ -200,7 +188,7 @@ contract calle_contract {
 
     assert_eq!(
         vm.logs,
-        "runtime_error: read integer out of bounds in test.sol:86:18-30,\n"
```

Comment:
Maybe it's just better to remove the line numbers and exact positions (I think we just want to ensure this error is present, there's no way it stems from a different place indicating a bug in the compiler)